firstapp/home/views.py

```diff
@@ -7,3 +7,12 @@
 def say_hello(request):
     return HttpResponse('Hello World Django')
 
+
+def browser_info(request):
+    user_agent = ''
+    if request.method == 'GET':
+        user_agent = request.GET.get('user_agent', '')
```

Comment:
!!! what is request.Get.get() !!!!

instead try request.META['HTTP_USER_AGENT'])

Reply:
I used this code and it worked. thank you

Reply:
I created the '.gitignore' and pushed it again